### ISSUE TITLE
Add various other things to print style sheet

### ISSUE THREAD

**Issue by ausmeyer** [PeerJ/paper-now#50]
It seems like the author affiliations, funding, contributions, acknowledgements, and corresponding author annotations should be added to the print style sheet.

**Comment by hubgit** [comment]
Yes, that's something that just hasn't been done yet (the print style sheet needs quite a bit of work, still).

Do they need any particular design, or just to be tidier than they are now?

**Comment by ausmeyer** [comment]
I actually like the way PeerJ does it on PDF typesetting. 

https://peerj.com/articles/988.pdf

I think this is the only barrier to being able to realistically submit a pdf directly for review to PeerJ.

**Comment by ausmeyer** [comment]
I should have said also. The only reason I'm thinking about the print style sheet is that it is relatively easy to print to a pdf and submit immediately to PeerJ that way.
